File: BotsSDK/korebotsdklib/src/main/java/kore/botssdk/models/BrandingQuickStartButtonButtonsModel.java
package kore.botssdk.models;

import androidx.annotation.NonNull;

import java.io.Serializable;

public class BrandingQuickStartButtonButtonsModel implements Serializable {
    private String title;
    private String description;

    @NonNull
    public String getDescription() {
        return description;
    }

    public void setDescription(@NonNull String description) {
        this.description = description;
    }

    private BrandingQuickStartButtonActionModel action;

    @NonNull
    public String getTitle() {
        return title;
    }

    @NonNull
    public BrandingQuickStartButtonActionModel getAction() {
        return action;
    }

    public void setTitle(@NonNull String title) {
        this.title = title;
    }

    public void setAction(@NonNull BrandingQuickStartButtonActionModel action) {
        this.action = action;
    }
}

File: BotSdk/botclient/src/main/java/com/kore/network/api/responsemodels/branding/BrandingQuickStartButtonButtonsModel.kt
package com.kore.network.api.responsemodels.branding

import com.google.gson.annotations.SerializedName

data class BrandingQuickStartButtonButtonsModel (
    @SerializedName("title")
    val title : String = "",
    @SerializedName("description")
    val description : String = "",
    @SerializedName("action")
    val action : BrandingQuickStartButtonActionModel
)